supervisor/ingress.py

```diff
@@ -35,6 +35,11 @@ def sessions(self) -> dict[str, float]:
         """Return sessions."""
         return self._data[ATTR_SESSION]
 
+    @property
+    def sessions_data(self) -> dict[str, float]:
```

Comment:
let's parse the session data from a dict into a DataClass (OOP Style) -> i.e. https://github.com/home-assistant/supervisor/blob/main/supervisor/discovery/__init__.py#L29 but today we can use data class (https://docs.python.org/3/library/dataclasses.html)